lib/phoenix/live_dashboard/page_builder.ex

```diff
@@ -298,6 +307,279 @@ defmodule Phoenix.LiveDashboard.PageBuilder do
+
+      def render_page(assigns) do
+        card(
+          title: "Run queues",
+          inner_title: "Total",
+          class: "additional-class",
+          value: 1.5
+        )
+      end
+
+  You can see it in use the Home and OS Data pages.
+
+  # Options
+
+  These are the options supported by the component:
+
+    * `:value` - Required. The value that the card will show.
+
+    * `:title` - The title above the card.
+      Default: `nil`.
+
+    * `:inner_title` - The title inside the card.
+      Default: `nil`.
+
+    * `:hint` - A textual hint to show close to the title.
+      Default: `nil`.
+
+    * `:inner_hint` - A textual hint to show close to the inner title.
+      Default: `nil`.
+
+    * `:class` - A list of additional css classes that will be added along banner-card class.
+      Default: `[]`.
+  """
+  @spec card(keyword()) :: component()
+  def card(assigns) do
+    assigns =
+      assigns
+      |> Map.new()
+      |> CardComponent.normalize_params()
+
+    {CardComponent, assigns}
+  end
+
+  @doc """
+  Renders a fields card component.
+
+  It can be rendered in any dashboard page via the `render_page/1` function:
+
+      def render_page(assigns) do
+        fields_card(
+          title: "Run queues",
+          inner_title: "Total",
+          class: "additional-class",
+          fields: ["USER": "...", "ROOTDIR: "..."]
+        )
+      end
+
+  You can see it in use the Home page in the Environment section.
+
+  # Options
+
+  These are the options supported by the component:
+
+    * `:fields` - Required. A list of key-value elements that will be shown inside the card.
+
+    * `:title` - The title above the card.
+      Default: `nil`.
+
+    * `:inner_title` - The title inside the card.
+      Default: `nil`.
+
+    * `:hint` - A textual hint to show close to the title.
+      Default: `nil`.
+
+    * `:inner_hint` - A textual hint to show close to the inner title.
+      Default: `nil`.
+  """
+  @spec fields_card(keyword()) :: component()
+  def fields_card(assigns) do
+    assigns =
+      assigns
+      |> Map.new()
+      |> FieldsCardComponent.normalize_params()
+
+    {FieldsCardComponent, assigns}
+  end
+
+  @doc """
+  Renders a column component.
+
+  It can be rendered in any dashboard page via the `render_page/1` function:
+
+      def render_page(assigns) do
+        page_columns(
+          columns: [
+            card(...),
+            card_usage(...)
+          ]
+        )
+      end
+
+  You can see it in use the Home page and OS Data pages.
+
+  # Options
+
+  These are the options supported by the component:
+
+    * `:columns` - Required. A list of components.
+      It can receive up to 3 components.
+      Each element will be one column.
+  """
+  @spec page_columns(keyword()) :: component()
+  def page_columns(assigns) do
+    assigns =
+      assigns
+      |> Map.new()
+      |> PageColumnsComponent.normalize_params()
+
+    {PageColumnsComponent, assigns}
+  end
```

Comment:
There is a bit of inconsistency between `page_columns` and `row`. Is `page_columns` prefixed with `page_` because it can only be used at the root? If so, should we document that? Also, we do `page_columns(columns: [...])` and `row(components: [...])`. Should it be called components for both?

Reply:
Initially I decide to use `page_columns` to avoid confusion with `columns/0` function used in tables. I change the latter to `table_columns/0 `and rename `page_columns` to `columns`